tests/framework/extensions/rbac/constants.go

```diff
@@ -0,0 +1,46 @@
+package rbac
+
+const (
+	Admin                            = "admin"
+	RestrictedAdmin                  = "restricted-admin"
+	StandardUser                     = "user"
+	RoleReadOnly                     = "read-only"
```

Comment:
Not used, please remove

Reply:
these are used in rbacuserroles > restrictedadmin_role_test.go, cluster_role_test.go